src/get-started/java-binding.md

```diff
@@ -501,11 +501,19 @@ Currently Java Binding includes the following built-in services:
   Time oracle allows user services to access the calendar time supplied by
   validator nodes to the blockchain.
 
-To enable a particular service, include its name in `ejb_app_services.toml`
-configuration file in the EJB App's directory with the following content:
+## Services Definition
+
+Services are disabled by default. To enable a particular service,
```

Comment:
I'd rewrite and shorten this paragraph somehow. @Maria-Nosyk , could you help with that please?

Maybe, "No services are enabled on the node by default. `ejb_app_services.toml` configuration file defines the services to enable. This configuration file should be located in a **working directory**, where you are running commands. It consists of two sections: `system_services` and `user_services`."